cosmos/providers/dbt/parser/project.py

```diff
@@ -64,7 +64,7 @@ def __post_init__(self) -> None:
         for base_node in ast.find_all(jinja2.nodes.Call):
             if hasattr(base_node.node, "name"):
                 # check we have a ref - this indicates a dependency
-                if base_node.node.name == "ref":
+                if base_node.node.name == "ref" or base_node.node.name == "source":
```

Comment:
This is actually the only change needed -- right now I have the user update the path to use `dbt-ol` via `dbt_args` like so:
```
dbt_args={
            "dbt_executable_path": "/usr/local/airflow/dbt_venv/bin/dbt-ol",
}
```
but we could add a lineage option in `dbt_args` directly that could look for an assumed location of this executable.

Reply:
@denimalpaca is there some form of test we can set up to make sure this doesn't break in the future?

Reply:
That which part doesn't break?

the conversation Chris H started about env should resolve the latter part (and I did implement `dbt-ol` automatically getting picked up when it exists in the env)